soh/src/overlays/gamestates/ovl_file_choose/z_file_choose.c

```diff
@@ -1785,6 +1791,20 @@ void FileChoose_Main(GameState* thisx) {
         gSaveContext.skyboxTime += 0x10;
     }
 
+    if (CVar_GetS32("gSkipLogoTitle", 0) && CVar_GetS32("gSaveFileID", 0) < 3) {
+        if (Save_Exist(CVar_GetS32("gSaveFileID", 0)) && FileChoose_IsSaveCompatible(Save_GetSaveMetaInfo(CVar_GetS32("gSaveFileID", 0)))) {
```

Comment:
So it looks like if we have an incompatible save file we set the CVar to 4 and don't set any menuMode or selectMode, how does that actually look in-game?

Reply:
Just loads you into the file select screen as if you already had 4 selected